packages/components/webpack.config.js

```diff
@@ -197,7 +197,6 @@ module.exports = {
             'babel-polyfill': 'babel-polyfill',
             'prop-types': 'prop-types',
             'react-transition-group': 'react-transition-group',
-            'tt-react-custom-scrollbars': 'tt-react-custom-scrollbars',
```

Comment:
More packages have `tt-react-custom-scrollbars` imported. Can we uninstall them from those too?

Reply:
Yeah, I think some of them are from `@deriv/deriv-charts` as it's still a dependency in the library, I'll try to see if I can find any packages within `deriv-app` that are still using it.